src/ReactiveUI/Activation.cs

```diff
@@ -315,16 +343,29 @@ static IDisposable handleViewModelActivation(IViewFor view, IObservable<bool> ac
 
     /// <summary>
     /// This class implements View Activation for classes that explicitly describe
-    /// their activation via ICanActivate. This class is used by the framework.
+    /// their activation via <see cref="ICanActivate"/>. This class is used by the framework.
     /// </summary>
     public class CanActivateViewFetcher : IActivationForViewFetcher
```

Comment:
I'm wary of breaking things, but the pre-existing comment is along the lines of "used by the framework" and it's explicitly registered by ReactiveUI, so think its both fairly low risk and fairly low impact....